travis-ymls/python-memcached.travis.yml

```diff
@@ -0,0 +1,43 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : python-memcached
+# Source Repo         : https://github.com/linsomniac/python-memcached
+# Travis Job Link     : https://travis-ci.com/github/santosh653/python-memcached
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+
+arch:
+  - amd64
+  - ppc64le
+  
+language: python
+python:
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.6
+  - pypy
+#Excluding pypy for power as it is not supported.
+jobs:
+ exclude:
+  - python: pypy
+    arch: ppc64le
+services:
+  - memcached
+before_install:
+  - |
+     if [[ "$TRAVIS_CPU_ARCH" = "ppc64le" ]]; then
+      sudo apt-get install memcached  
+     fi
+install: python setup.py install
+before_script: 
+  - python -m pip install -U pip
+  - python -m pip install flake8
```

Comment:
whats the reason "pip install -r test-requirements.txt" from original file removed ?